5.5/Dockerfile

```diff
@@ -37,6 +36,13 @@ RUN rpmkeys --import file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7 && \
     test "$(id mysql)" = "uid=27(mysql) gid=27(mysql) groups=27(mysql)" && \
     chmod o+w -R /var/lib/mysql
 
+# When bash is started non-interactively, to run a shell script, for example it
+# looks for this variable and source the content of this file. This will enable
+# the SCL for all scripts without need to do 'scl enable'.
+ENV BASH_ENV=/var/lib/mysql/scl_enable \
+    ENV=/var/lib/mysql/scl_enable \
+    PROMPT_COMMAND=". /var/lib/mysql/scl_enable"
+
```

Comment:
@bparees I want to have comment on this to explain why we are doing it... squashing should take care of this